### PR TITLE
feat(St-Page-Title): Add edit title

### LINKED ISSUE #261: Page Title title should be editable
** Please don't delete this template or we'll close your issue **

**I'm submitting a ...**
- [ ] bug report
- [ X ] feature request
- [ ] question

**Define your petition as much as possible**

In Page Title component, 'title' input should be an editable field.

**What is the current behavior?**

Title input can't be edited

**If the current behavior is a bug, please provide the steps to reproduce it**

**What is the expected behavior?**

**What is the motivation / use case for changing the behavior?**

UX requirement

**Please tell us about your environment:**

- Egeo version: 2.1.0
- Browser: [all ]

**Other information** (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, eg. stackoverflow, gitter, etc)

### PR DESCRIPTION
**Pull request by pjpenast**
## TYPE
- New feature in component

## Description
Add editable option in st-page-title

close #261 

### Documentation
- [x] Have explained API (Inputs, Outpus, Models, Functions, etc)
- [x] Have example running
- [x] Have example code (HTML, ts)
- [x] Have changelog.md updated

### Code
- [x] Have 3 spaces indentation
- [x] Pass ts lint Task
- [x] Pass Sass lint Task
- [x] Have QA-tags

### Tests
- [x] Have 70% Tests coverage

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,7 +43,7 @@
    "dependencies": {
       "angular2-virtual-scroll": "0.1.5",
       "lodash": "4.17.4",
-      "popper.js": "1.9.9",
+      "popper.js": "^1.9.9",
```
Comment (ppenalver) [inline package.json]: remove ^

File: package.json [hunk 2]
```diff
@@ -58,7 +58,7 @@
       "@angular/platform-browser-dynamic": "4.1.2",
       "@angular/platform-server": "4.1.2",
       "@angular/router": "4.1.2",
-      "@stratio/egeo-theme": "0.10.0",
+      "@stratio/egeo-theme": "^0.10.0",
```
Comment (ppenalver) [inline package.json]: remove ^

File: src/lib/directives/st-input-adjustable.ts [hunk 3]
```diff
@@ -0,0 +1,81 @@
+import { ARGUMENT_TYPES } from 'tslint/lib/rules/completedDocsRule';
```
Comment (ppenalver) [inline src/lib/directives/st-input-adjustable.ts]: ???

**Comment by pjpenast** [comment]
@ppenalver Add test, review again please

**Pull request merged**